frontend/src/components/Button/Button.styled.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+export const StyledButton = styled.button`
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  height: 30px;
+  width: 100px;
+  padding: 15px;
+  border-radius: 5px;
+  border: none;
+  color: #ffffff;
+  background-color: #1a8cd8;
+  box-shadow: 0px 4px 4px 0px #00000040;
```

Comment:
Co to za kolorek? Chyba za dużo tych zer?

Reply:
![image](https://user-images.githubusercontent.com/88898014/151007118-7c706bec-db76-426f-9110-35229d27cc3b.png)

czarny z bardzo małym opacity

Reply:
Kolory z figmy są :)